source/sync/data-model/data-model-map.txt

```diff
@@ -249,6 +249,8 @@ structure.
 For more information on modeling mixed properties in
 an App Services schema, refer to :ref:`Mixed - Schema Types <schema-type-mixed>`.
 
+.. _collections-in-mixed:
+
 Collections in Mixed Properties
```

Comment:
per [DOCSP-40333](https://jira.mongodb.org/browse/DOCSP-40333), can we also get the `/includes/note-collections-in-mixed-sync.rst` note updated with the Swift release version (v10.15.0)? Or are you doing that in a separate PR?

Reply:
I'll update in this PR, thanks!